<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="e_address"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_port">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="b_connect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_selectFile">
      <property name="text">
       <string>Select file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_send">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_status">
        <property name="text">
         <string>Not connected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>178</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
